### ISSUE TITLE
[issue] Why was the validation set used for testing and not the training set? Is that right?

### ISSUE THREAD

**Issue by STHxiao** [SegmentationBLWX/sssegmentation#55]

**Comment by CharlesPikachu** [comment]
The division of the data set is defined by the authors of each data set. The previous papers also conducted experiments according to these splits.

**Comment by STHxiao** [comment]
Thank you very much !
